dev-qt/qtwebengine/qtwebengine-5.15.9_p20230505.ebuild

```diff
@@ -42,7 +42,7 @@ RDEPEND="
 	dev-libs/libevent:=
 	dev-libs/libxml2[icu]
 	dev-libs/libxslt
-	dev-libs/re2:=
+	<dev-libs/re2-2023-06-02:=
```

Comment:
Could you add a comment noting what you said on IRC wrt the C++ standard versions?